packages/autocomplete-plugin-recent-searches/src/createPlugin.ts

```diff
@@ -0,0 +1,66 @@
+import { AutocompletePlugin } from '@algolia/autocomplete-core';
+
+import { createRecentSearchStore } from './createRecentSearchStore';
+
+type PluginOptions = {
+  limit?: number;
```

Comment:
https://github.com/algolia/autocomplete.js/pull/316/commits/e0e1749141bd8cc42a8ce9dd22823394fe3c4250

but should it be a mandatory param? If they use only one instance of autocomplete (which might be a common use case), they don't need to care.

Reply:
We should make it mandatory at first because as soon as they decide to use another autocomplete, it'll screw up.

It's better to have a strict API at first and then to loosen it, because the other way around is not possible.

The only case where it's not required is if we're able to generate IDs for each Autocomplete instance, but I don't think we can do this.

Reply:
It makes sense!

Reply:
done https://github.com/algolia/autocomplete.js/pull/316/commits/90fc79061c0beeb95cceedd446d8f1e2445a8b21